k2/python/csrc/torch/mutual_information_cpu.cu

```diff
@@ -103,7 +104,7 @@ torch::Tensor MutualInformationCpu(torch::Tensor px, torch::Tensor py,
           }
           for (int t = t_begin + 1; t <= t_end; ++t)
             p_a[b][s_begin][t] =
-                p_a[b][s_begin][t - 1] + py_a[b][s_begin][t - 1];
+                p_a[b][s_begin][t - 1] + py_scale * py_a[b][s_begin][t - 1];
```

Comment:
... BTW, I think we can leave the core, C++ part of the code as it is, because this can be simulated by just modifying the expression for py, including the px part in it.  Again, I don't recall the exact offsets involved but it should be fairly simple.

Reply:
Ok, thanks, I will try to figure it out.